packages/hardhat/contracts/strategy/CrabMigration.sol

```diff
@@ -120,6 +120,7 @@ contract CrabMigration is Ownable {
         weth = WETH9(_weth);
         dToken = _dToken;
         wPowerPerp = crabV1.wPowerPerp();
+        WETH9(_weth).approve(_eulerMainnet, type(uint256).max);
```

Comment:
can we just use `weth` here instead of reinitializing `WETH9(_weth)`?

Reply:
We can't use any variables that are in the constructor for anything. So we have to use the params passed in (why using _eulerMainnet as well instead of eulerMainnet)